RadarSDK/Radar.m

```diff
@@ -1162,7 +1162,7 @@ - (void)dealloc {
 
 + (BOOL)isTestKey {
     NSString *publishableKey = [RadarSettings publishableKey];
-    if ([publishableKey hasPrefix:@"prj_test_pk"] || [publishableKey hasPrefix:@"org_test_pk"]) {
+    if ([publishableKey hasPrefix:@"prj_test_pk"] || [publishableKey hasPrefix:@"org_test_pk"] || [RadarSettings userDebug]) {
```

Comment:
This is good for consistency with Android, maybe in a future version we pull this out of the `isTestKey` function

Reply:
SG.